README.md

```diff
@@ -1,40 +1,67 @@
 # evmosjs
 
-JS and TS libs for Evmos
+JS and TS libs for Evmos.
 
-## Example
+## Installation
 
-### Get account information
+evmosjs uses [buf.build](https://buf.build/) to manage Protobuf dependencies. To install evmosjs packages in your project,
+follow the instructions corresponding to your package manager.
 
-Get the account number, sequence and pubkey from an address.
-NOTE: if the address had not sent any transaction to the blockchain, the pubkey value are going to be empty.
+### NPM
+
+Add the following line to an `.npmrc` file in your project root:
+
+```ini
+@buf:registry=https://buf.build/gen/npm/v1
+```
+
+Then run `npm install evmosjs` (or `npm install @evmosjs/[package]`).
```

Comment:
nit, prefer code blocks for installation instructions since they are easier to copy